src/main/java/org/folio/service/LoanEventsGenerationService.java

```diff
@@ -43,9 +47,9 @@ public LoanEventsGenerationService(Map<String, String> okapiHeaders, Vertx vertx
 
   @Override
   protected Future<Loan> generateEvents(Loan loan) {
-    final String loanId = loan.getId();
-    log.info("Generating events for loan {}...", loanId);
+    log.debug("generateEvents:: parameters loan: {}", () -> logAsJson(loan));
 
+    final String loanId = loan.getId();
     userIds.add(loan.getUserId());
 
     return succeededFuture(loan)
```

Comment:
add method name to onSuccess/onFailure and log result itself

Reply:
done